packages/diagrams/frontend/sirius-components-diagrams/src/renderer/dropNode/useDropNode.ts

```diff
@@ -163,7 +166,8 @@ export const useDropNode = (): UseDropNodeValue => {
     const dropDataEntry: GQLDropNodeCompatibility | undefined = diagramDescription.dropNodeCompatibility.find(
       (entry) => entry.droppedNodeDescriptionId === (computedNode as Node<NodeData>).data.descriptionId
     );
-    const compatibleNodes = getNodes()
+    const compatibleNodes = reactFlowInstance
+      .getNodes()
```

Comment:
Couldn't we be more consistent and never use the `reactFlowInstance` variable and always deconstruct it into `getNodes`, `getEdges` ...
Perhaps even rename these methods to make it clearer that these are react flow's internal values.